theme_bewise/static/src/scss/primary_variables.scss

```diff
@@ -101,23 +98,16 @@ $o-theme-font-configs: (
 
 $o-website-values-palettes: (
     (
-        'color-palettes-name': 'bewise-1',
+        'color-palettes-name': 'default-19',
 
-        'header-font-size': (20 / 16) * 1rem,
-        'font-size-base': 1.3125rem,
-
-        'font': 'Darker Grotesque',
-        'headings-font': 'Raleway',
```

Comment:
Do you think the `Raleway` should be removed here (since it's no longer used) or you prefer to keep it available ?

Reply:
I think that it should remain available.
I people want to go back to this font it would be handy to find it directly in the list